patient/README.md

```diff
@@ -0,0 +1,124 @@
+# Patient Mapping
+
+## XML HL7 > JSON FHIR
+
+A Patient resource is mapped from a HL7 Patient.
+
+| Mapped to (JSON FHIR Immunization field) | Mapped from (XML HL7 / other source)                                                                                                                                     |
+|------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| id                                       | set value = Generated GUID                                                                                                                                               |
+| meta.profile\[0]                         | fixed value = `"https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-Patient-1"`.                                                                                |
+| meta.versionId                           | fixed value = `1521806400000`                                                                                                                                            |
+| indentifier\[0].system                   | fixed value = `https://fhir.nhs.uk/Id/nhs-number`                                                                                                                        |
+| indentifier\[0].value                    | `Patient / id [@extension]`                                                                                                                                              |
+| managingOrganization                     | referenced to mapped AgentOrganization: [Organization](https://github.com/NHSDigital/patient-switching-adaptors-mapping-documentation/blob/main/organisations/README.md) |
+
+## Unmapped fields
+
+ - implicitRules
+ - language
+ - text
+ - contained
+ - extension
+ - modifierExtension
+ - identifier.use
+ - identifier.type
+ - identifier.type.coding
+ - active
+ - name
+ - name.period
+ - name(official)
+ - telecom
+ - telecom.period

```

Comment:
we're not mapping `telecom`, so we don't need to say we're not mapping `telecom.period`